docs/technical_reference/unit_models/bipolar_electrodialysis_0D.rst

```diff
@@ -0,0 +1,369 @@
+     "transport number of ions in the membrane phase", ":math:`t_j`", "ion_trans_number_membrane", "['bpem'], ['Na\ :sup:`+`', '\Cl\ :sup:`-`', 'H\ :sup:`+`', 'OH\ :sup:`-`']", "dimensionless", 4
+
+**Note**
+ :sup:`1` DOF number takes account of the indices of the corresponding parameter.
+
+ :sup:`2` A user should provide either current or voltage as the electrical input, in correspondence to the "Constant_Current" or "Constant_Voltage" treatment mode
+
+
+Solution component information
+------------------------------
+To fully construct solution properties, users need to provide basic component information of the feed solution to use this model. Below is a sample:
+
+.. code-block::
+
+   ion_dict = {
+        "solute_list": ["Na_+", "Cl_-", "H_+", "OH_-"],
+        "mw_data": {
+            "Na_+": 23e-3,
+            "Cl_-": 35.5e-3,
+            "H_+": 1e-3,
+            "OH_-": 17.0e-3,
+        },
+        "elec_mobility_data": {
+            ("Liq", "Na_+"): 5.19e-8,
+            ("Liq", "Cl_-"): 7.92e-8,
+            ("Liq", "H_+"): 36.23e-8,
+            ("Liq", "OH_-"): 20.64e-8,
+        },
+        "charge": {"Na_+": 1, "Cl_-": -1, "H_+": 1, "OH_-": -1},
+        "diffusivity_data": {
+            ("Liq", "Na_+"): 1.33e-9,
+            ("Liq", "Cl_-"): 2.03e-9,
+            ("Liq", "H_+"): 9.31e-9,
+            ("Liq", "OH_-"): 5.27e-9,
+        },
+    }
+
+This model, by default, uses H\ :sub:`2`\ O  as the solvent of the feed solution. Please note that H\ :sup:`+` and OH\ :sup:`-` information must be supplied. Otherwise an error will be thrown.
+
+Information regarding the property package this unit model relies on can be found here: 
+
+:py:mod:`watertap.property_models.ion_DSPMDE_prop_pack`
+
+Operation without catalyst
+--------------------------
+
+The simplest water splitting mode is without any catalyst. Hence default the config ``has_catalyst`` is set to false. The Mass balance equations are summarized in **Table3**. Further details on these can be found in the *References*.
+
+.. csv-table:: **Table 3** Mass Balance Equations
+   :header: "Description", "Equation", "Index set"
+
+   "Component mass balance", ":math:`N_{j, in}^{acidic \: or\:  basic}-N_{j, out}^{acidic\: or\:  basic}+J_j^{acidic\: or\:  basic} bl=0`", ":math:`j \in \left['H_2 O', '{Na^+} ', '{Cl^-} '\right]`"
+   "mass transfer flux, basic, solute", ":math:`J_j^{C} = -t_j^{bpem}\frac{\xi i_{lim}}{ z_j F}`", ":math:`j \in \left['{Na_+} ', '{Cl^-} '\right]`"
+   "mass transfer flux, acidic, proton", ":math:`J_j^{C} = \frac{i - i_{lim}}{F}`", ":math:`j \in \left['{H^+} '\right]`"
+   "mass transfer flux, acidic, hydroxide", ":math:`J_j^{C} = 0`", ":math:`j \in \left['{OH^-} '\right]`"
+   "mass transfer flux, basic, proton", ":math:`J_j^{C} = 0`", ":math:`j \in \left['{H^+} '\right]`"
+   "mass transfer flux, basic,  hydroxide", ":math:`J_j^{C} = \frac{i - i_{lim}}{F}`", ":math:`j \in \left['{OH^-} '\right]`"
```

Comment:
If there is a flux of H+ and OH- , why are they not reflected in the mass balance eq?  I also think you should show a reaction term, a critical feature of an electrolysis cell, in the overall mass balance .  The namely "mass transfer" terms for H+ and OH- are essentially reaction terms by your equations, although their mass was moved but usually they are referred as reaction terms.